core/commands.go

```diff
@@ -552,6 +552,12 @@ var (
 		Arity:    -2,
 		KeySpecs: KeySpecs{BeginIndex: 1},
 	}
+	dbSizeCmdMeta = DiceCmdMeta{
+		Name:  "DBSIZE",
+		Info:  `DBSIZE Return the number of keys in the currently-selected database`,
+		Eval:  evalDBSIZE,
+		Arity: -1,
```

Comment:
Arity should be 1 .
you check that in  [Redis](https://github.com/redis/redis/blob/2b88db90aad82574541edae6329ef24053fb4188/src/commands/dbsize.json#L7)